<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransportDiscontinued</class>
 <widget class="QWidget" name="TransportDiscontinued">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Routes</string>
  </property>
  <widget class="QListView" name="lv_availableRoutes">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>391</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Available Routes</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>321</width>
     <height>351</height>
    </rect>
   </property>
   <property name="title">
    <string>Route Details</string>
   </property>
   <widget class="QComboBox" name="cb_Origin">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>21</y>
      <width>82</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Origin Hub</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_Destination">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>53</y>
      <width>108</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Destination Hub</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>80</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Transport Type</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_TransportType">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_AddEvent">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Event</string>
    </property>
   </widget>
   <widget class="QLabel" name="errorLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
